source/api/commands/route.md

```diff
@@ -191,6 +195,10 @@ cy.route('POST', '**/comments', commentsResponse)
 
 Any request that matches the `method` and `url` of a route will be responded to based on the configuration of that route.
 
+{% note bolt %}
+`GET` is the default HTTP method used to match routes. If you want to stub a route with another HTTP method such as `POST` then you {% urlHash 'must be explicit about the method' Arguments %}.
+{% endnote %}
+
 If a request doesn't match any route, {% urlHash "it will automatically receive a 404" Notes %}. For example, given we have the following routes:
```

Comment:
Well... darn 😢 

Only just spotted this line about _"automatically [receiving] a 404"_... I'd better go back and amend my earlier PR #876